static/library/line_chart_component.ts

```diff
@@ -86,6 +86,12 @@ export class DatacommonsLineComponent extends LitElement {
   // Statistical variable DCIDs
   @property({ type: Array<string>, converter: convertArrayAttribute })
   variables!: Array<string>;
+  
+  // Optional: Regex used to process variable names
+  // If provided, will only use the first case of the variable name that matches
+  // this regex.
```

Comment:
can you add an example regex and the before/after of what happens to the variable to this comment

Reply:
done